recipes/parsnip-cif/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "parsnip-cif" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "e957191dfb0c96ee23dfbfa1e66b6c45784d48188da73dc66a5a5aeb11d25717" %}
```

Comment:
Please don't use excess jinja variable. It makes the recipe harder to read and can break the updating bot.